test/Chirp.Razor.Tests/ChirpTests.cs

```diff
@@ -210,7 +190,7 @@ public void TestGetCheepsFromAuthorWithID(string authorId)
         // Arrange
         
         // Act
-        var result = _cheepRepo.ReadCheepsByID(authorId);
+        var result = _cheepService.GetCheepsFromAuthorByID(authorId);
         // Assert
         Assert.NotEmpty(result);
```

Comment:
Maybe we need to change assert to check if the result is actually written by the author